### PR TITLE
Us 337 - API Docs

### PR DESCRIPTION
**Pull request by CBrobsto**
## Objective
This branch adds basic API documentation to the API folder in a markdown file that is viewable on GitHub like a ReadMe file. 

## How to Test (if applicable)
You can either view the file for accuracy in GitHub or you can copy/paste the contents here: https://dillinger.io/

## Requirements
https://tree.taiga.io/project/cbrobsto-iron-db/us/337?milestone=260360

### CODE REVIEW

File: API/controller/app.js [hunk 1]
```diff
@@ -113,7 +113,11 @@ app.use(express.static(path.join(__dirname, "../public")));
 
 app.use(
   session({
+<<<<<<< HEAD
+    secret: "j2xOm%iKqFOqqNoJ#6DUlWmP(*JVIe!V",
+=======
     secret: "(@6UiSfotyHSp8HdC0N7OjKCX*7HtMyO",
+>>>>>>> c7b2459499e31c517bf5af3b121f5ccd7a4a6b75
```
Comment (mjidris) [inline API/controller/app.js]: You'll want to remove these merge conflict markers.

File: API/controller/db/entry-parser.js [hunk 2]
```diff
@@ -248,7 +248,7 @@ function convertUnitString(originalUnit) {
   return units;
 }
 
-
+/** */
```
Comment (mjidris) [inline API/controller/db/entry-parser.js]: ?

File: API/documentation.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
+# **API Reference**
+***********************
+## Part 1: Endpoints
+## All endpoints:
```
Comment (mjidris) [inline API/documentation.md:4]: Awesome 🙌

**Review by mjidris — CHANGES_REQUESTED** [review]
Overall looks great, you'll want to remove the merge conflict marker before merging it in.

**Review by StellaJaneRasmussen — APPROVED** [review]
Fantastic job! There are a few merge conflicts, but those aren't related to the documentation itself, so I give my approval.

File: API/controller/app.js (hunk 1 above)
Comment (StellaJaneRasmussen) [reply API/controller/app.js]: +1

File: API/documentation.md (hunk 3 above)
Comment (CBrobsto) [reply API/documentation.md:4]: Thank you!

**Review by CBrobsto — COMMENTED** [review]
I fixed the conflict markers and removed the empty comment.

**Review by mjidris — APPROVED** [review]
💯

**Pull request merged**